pennylane/__init__.py

```diff
@@ -133,6 +136,63 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, name: str):
+    """load(name, quantum_circuit_object)
+    Load a plugin :func:`~.load` which can then be used to convert objects of
+    quantum circuits from other frameworks.
+
+    Args:
+        quantum_circuit_object: the quantum circuit that will be converted
+            to a PennyLane template
+        name (str): the name of the plugin to convert from
+
+    Returns:
+        _function: the PennyLane template created from the quantum circuit
+            object
+    """
```

Comment:
We should have a `.. note::` in this docstring, linking to the `pennylane.ai/plugins.html` page, and telling the user to check out the available plugins and their documentation for more details on what conversions are supported.